students/philip_behrend/lesson01/test_unit.py

```diff
@@ -0,0 +1,27 @@
+from inventory_management.product_classes import Inventory, Furniture, ElectricAppliances
+import inventory_management.main
+from unittest import TestCase 
+
+class InventoryTests(TestCase):
+    """Test for the Inventory class"""
+
```

Comment:
Use _setUp_ to create your instance of _Inventory_, which you can then reuse across _InventoryTests_. Note that it's being created as an instance variable of _InventoryTests_, hence the use of _self.inv_ instead of just _inv_, that way it can be accessed by each test.
```
    def setUp(self):
        self.inv = Inventory('125', "Test Item", '50', '80')
```
Each test, in turn, will need to to also refer to it as _self.inv_, for example:

```
    def test_inv_init(self):
        inv = Inventory(125, 'Test Item', 50, 80)
        self.assertEqual(self.inv.product_code, 125)
```